src/ImGui.NET/ImGuiNative.cs

```diff
@@ -272,7 +272,7 @@ public static unsafe class ImGuiNative
         public static extern void igTextWrapped(string fmt);
 
         [DllImport(cimguiLib)]
-        public static extern void igTextUnformatted(byte* text, byte* text_end);
+        public static extern void igTextUnformatted(string text, string text_end);
```

Comment:
Also, these range functions are pretty much useless in c#, maybe omit them completely?